src/viur_cli/conf.py

```diff
@@ -2,87 +2,15 @@
 Everything related to maintaining the project.json configuration file for a viur project.
 """
 
-import click
-import os
 import json
 from .utils import *
-from pathlib import Path
 
 DEFAULT_PYODIDE_VERSION = "0.19.1"
 
-projectConfig = None
+pectConfig = None
```

Comment:
This looks like typo

```suggestion
projectConfig = None
```